py2pack/version.py

```diff
@@ -16,6 +16,6 @@
 # limitations under the License.
 
 # this one is read by hatch or hatchling during build
-__version__ = "0.9.0"
```

Comment:
Why do you feel the need to create new release here? What this has to do with `fix bug: AttributeError …`?

Reply:
because of new feautures?

Reply:
Please, keep any PR only to the topic it purports to resolve. Creating new releases etc. is not a job of this PR.